name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesPerformanceView.java

```diff
@@ -123,7 +123,7 @@ public FilterDropDown(IPreferenceStore preferenceStore)
 
             addDisposeListener(e -> {
                 preferenceStore.setValue(SecuritiesPerformanceView.class.getSimpleName() + "-sharesGreaterZero", //$NON-NLS-1$
-                                recordFilter.contains(sharesGreaterZero));
+                                recordFilter.contains(sharesNotZero));
```

Comment:
Similar issue as in the other PR: I’m not changing the key for this setting even though “sharesGreaterZero” is no longer accurate, so that existing settings will still load.

Reply:
I assume labels will reflect logic `Shares held <>0`

Reply:
You might want to look at the diff instead of assuming anything …

Reply:
I scanned the diffs, didn't see it (it might be there) and noticed comment about other PR. If you expect more research than that for a quick comment you'll be disappointed.